src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -242,34 +244,34 @@ const loadJobs = (specifiedVc) => {
       return username + '-' + name;
     },
     'columns': [
-      {title: 'Job', data: null, render({legacy, name, namespace, username}, type) {
-        if (type !== 'display') return name;
+      {title: 'Job', data: null, responsivePriority: 1, render({legacy, name, namespace, username}, type) {
+        if (type !== 'display') return ellipsis(name);
```

Comment:
Do not use ellipsis here as it's used for search/sort
```suggestion
        if (type !== 'display') return name;
```

BTW, have you tried using CSS ellipsis instead of hardcoded character width?

```css
el {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}
```

Reply:
fixed